prboom2/src/g_game.c

```diff
@@ -224,23 +224,6 @@ static const struct
   { wp_bfg,          wp_bfg }
 };
 
-// HERETIC_TODO: dynamically set these
-static const struct
-{
-    weapontype_t weapon;
-    weapontype_t weapon_num;
-} heretic_weapon_order_table[] = {
```

Comment:
I'm surprised this is unused.  Missing branches on game elsewhere?

Reply:
This stays

Reply:
What should use it, then?